Code/Framework/AzCore/Tests/Serialization.cpp

```diff
@@ -1752,11 +1752,12 @@ namespace UnitTest
         IO::ByteContainerStream<AZStd::vector<char>> stream(&buffer);
         ASSERT_TRUE(Utils::SaveObjectToStream(stream, DataStream::ST_XML, &testType, m_serializeContext.get()));
 
-        constexpr const char* expectedValue = R"(<ObjectStream version="3">
-	<Class name="ElementOverrideType" type="{BAA18B6C-3CB3-476C-8B41-21EA7CE1F4CF}">
-		<Class name="int" field="field" value="1" type="{72039442-EB38-4D42-A1AD-CB68F7E0EEF6}"/>
-	</Class>
-</ObjectStream>)";
+        constexpr const char* expectedValue =
+            R"(<ObjectStream version="3">)" "\n"
+            "\t" R"(<Class name="ElementOverrideType" type="{BAA18B6C-3CB3-476C-8B41-21EA7CE1F4CF}">)" "\n"
+            "\t\t" R"(<Class name="int" field="field" value="1" type="{72039442-EB38-4D42-A1AD-CB68F7E0EEF6}"/>)" "\n"
+            "\t" R"(</Class>)" "\n"
```

Comment:
Why does this need tabs?

Reply:
The serializer outputs tabs and I'm doing a string comparison of the two so they have to match up.

Reply:
StripWhiteSpace didn't get rid of it?

Reply:
That only removed the whitespace at the beginning/end of the string